src/can.c

```diff
@@ -42,6 +52,57 @@ static const struct isotp_fc_opts fc_opts = {
     .stmin = 1, /* minimum separation time = 100 ms */
 };
 
+#ifdef CONFIG_THINGSET_CAN_PACKETIZED_REPORTS_RX
+struct thingset_can_rx_context
+{
+    uint8_t src_addr;
+    uint8_t seq;
+    bool escape;
+};
+
+NET_BUF_POOL_DEFINE(thingset_can_rx_buffer_pool, CONFIG_THINGSET_CAN_NUM_RX_BUFFERS, CONFIG_THINGSET_CAN_RX_BUF_PER_SENDER_SIZE, sizeof(struct thingset_can_rx_context), NULL);
```

Comment:
I've just realized that the automatic coding style check was not enabled. I've fixed this in main now. Could you please rebase again to include that check in CI?

And I recommend to enable clang-format check in your code editor, so that it fixes any format issues automatically based on the `.clang-format` file.

For VS Code that would be the following lines in `.vscode/settings.json`:

```json
{
    "editor.formatOnSave": true,
    "editor.formatOnSaveMode": "file"
}
```